packages/styles/scss/patterns/sub-patterns/card/index.scss

```diff
@@ -12,8 +12,8 @@
 @import 'carbon-components/scss/components/tile/tile';
 @import 'carbon-components/scss/components/link/link';
 
-@mixin card-link {
-  .#{$prefix}--card-link {
+@mixin card {
+  .#{$prefix}--card {
```

Comment:
Seeing a couple visual issues we need to look into.

<img src="https://user-images.githubusercontent.com/3793636/74188054-54dbc500-4c13-11ea-82d9-67f9edd2c176.png" alt="" width="300px" />

<img src="https://user-images.githubusercontent.com/3793636/74188146-7b016500-4c13-11ea-9806-b2cca029bf58.png" width="300px" alt="" />